src/main/scala/beam/utils/mapsapi/googleapi/GooglePolylineDecoder.scala

```diff
@@ -0,0 +1,52 @@
+package beam.utils.mapsapi.googleapi
+
+import scala.collection.mutable.ArrayBuffer
```

Comment:
@carloscaldas could we replace this with an existing api call instead of introducing this code?

Reply:
Yes. we can